test/utils/errors.js

```diff
@@ -0,0 +1,45 @@
+// Used for generic compiler inserted panics.
+const COMPILER = 0x00;
```

Comment:
assuming these were not defined anywhere in ethers

great work on having them here